changelog/fix-6474-consistent-dispute-naming

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Dispute reasons inconsistencies have been fixed.
```

Comment:
Should we mention that this includes a change to order notes, perhaps as an additional changelog entry?
```suggestion
Show descriptive dispute reasons in order notes.
```